templates/beegfs-oss-tuning.sh.j2

```diff
@@ -5,7 +5,7 @@ set -x
 
 echo always > /sys/kernel/mm/transparent_hugepage/enabled
 echo always > /sys/kernel/mm/transparent_hugepage/defrag
-for dev in {{ beegfs_oss | map(attribute='dev') | map('relpath', '/dev/') | join(' ') }}
+for dev in {{ beegfs_oss_tunable }}
 do
   echo mq-deadline > /sys/block/${dev}/queue/scheduler
```

Comment:
Can we improve resilience here by adding a check along the lines of ``test -e  /sys/block/${dev}``